src/elements/form/Control.ts

```diff
@@ -39,52 +39,52 @@ import { Helpers } from './../../utils/Helpers';
                         <div class="novo-control-input {{control.controlType}}" [ngSwitch]="control.controlType" [attr.data-automation-id]="control.key">
                             <!--Text-based Inputs-->
                             <!--TODO prefix/suffix on the control-->
-                            <div class="novo-control-input-container novo-control-input-with-label" *ngSwitchCase="'textbox'">
+                            <div class="novo-control-input-container novo-control-input-with-label" *ngSwitchCase="'textbox'" [tooltip]="control.tooltip" [tooltipPosition]="control.tooltipPosition">
```

Comment:
Can you put this on the NovoFormControl (we need to do some re-naming) at https://github.com/amrutha-rajiv/novo-elements/blob/52e1d708872034b8d7550cf234402cd14bf9cb4a/src/elements/form/FormUtils.ts#L33? Then grab it via `form.controls[control.key].tooltip` and that. That way FieldInteractions can update the tooltip if needed.